gensim/models/sent2vec.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+import logging
+import numpy as np
+from numpy import dot
+from gensim import matutils
+from random import randint
+import sys
+import random
+from gensim.utils import SaveLoad, tokenize
+import time
+
+logger = logging.getLogger(__name__)
+# TODO: add logger statements instead of print statements
+# TODO: add docstrings and tests
```

Comment:
It's time to make resolve this TODO's, start from `logger` and tests.

Reply:
Done. Kindly verify in the current commit.